app/models/intake/ctc_intake.rb

```diff
@@ -308,6 +312,11 @@ class Intake::CtcIntake < Intake
   has_one :bank_account, inverse_of: :intake, foreign_key: :intake_id, dependent: :destroy
   accepts_nested_attributes_for :bank_account
 
+  belongs_to :primary_drivers_license, class_name: "DriversLicense", optional: true
+  belongs_to :spouse_drivers_license, class_name: "DriversLicense", optional: true
+  accepts_nested_attributes_for :primary_drivers_license
```

Comment:
i think we can inline all of the accepts_nested_attributes things into one line with multiple arguments, if we want to